src/typography/text/Text.tsx

```diff
@@ -0,0 +1,50 @@
+import { css } from '@emotion/css';
+import * as React from 'react';
+import { Props } from './Props';
+import { getFontSize, getFontWeight, getThemeColor } from './utils';
+
+export function Text({ theme, size, color, align, text }: Props) {
+  return (
+    <div className={textStyle(align, size)} style={{ color: getThemeColor(theme, color) }}>
+      {size === 'Header1' ? (
+        <h1>{text}</h1>
+      ) : size === 'Header2' ? (
+        <h2>{text}</h2>
+      ) : size === 'Header3' ? (
+        <h3>{text}</h3>
+      ) : size === 'Header4' ? (
+        <h4>{text}</h4>
+      ) : (
+        <p>{text}</p>
+      )}
+    </div>
+  );
+}
```

Comment:
이건 return 문 안에서 4중 삼항연산자를 넣기보단 객체를 응용하면 좋을 것 같아요

```suggestion
export function Text({ theme, size, color, align, text }: Props) {
  const textWrapper = {
      header1: <h1>{text}</h1>,
      header2: <h2>{text}</h2>,
      header3: <h3>{text}</h3>,
      header4: <h4>{text}</h4>,
   } [size] || <p>{text}</p>;

  return (
    <div className={textStyle(align, size)} style={{ color: getThemeColor(theme, color) }}>
      {textWrapper}
    </div>
  );
}
```

Reply:
그러게용 기존꺼를 그대로 리펙토링 하다보니 이렇게 했네요 ㅋㅋ 수정해볼게요

Reply:
cc @k1a11220

Reply:
와우~